tests/Event/Http/Psr7BridgeTest.php

```diff
@@ -31,6 +31,24 @@ public function test I can create a response from a PSR7 response()
         ], $response->toApiGatewayFormat());
     }
 
+    public function test request with basic auth contains a user and password()
```

Comment:
Could you add these tests instead to the `CommonHttpTest` class? That way they are applied across function and fpm layers.

Which brings me to another request (sorry!): could you ensure the FPM layer has the same behavior (basic auth)? Because I guess the same problem exists with that layer as well, right?

Reply:
As sorry I meant that the FPM layer should behave the same way as well (`PHP_AUTH_USER` and `PHP_AUTH_PW` defined).

I can try to send a pull request today to add that if you want (PR on your branch).

Reply:
Sorry, misunderstood you.

Pushed a fix now. If that's now what you meant, please PR :)

Reply:
👍  I'm working on a PR to adjust some details in the tests, but just saw the last commit, that should definitely help thanks!

Reply:
🤔  actually while adding tests for this behavior, the tests pass without your last commit.

In retrospect, that makes sense now: the FPM process is automatically processing the HTTP Auth header and adding the variables manually. So we don't have to add code to support that. I'll remove those lines in my PR.

Reply:
OK we should be good, the PR is here: https://github.com/georgeboot/bref/pull/1